Dashboard/app.py

```diff
@@ -1,9 +1,14 @@
-from flask import Flask, render_template, url_for, Response
-import time
+from flask import Flask, render_template, request, redirect, url_for, Response
+from datetime import datetime
+from werkzeug.utils import secure_filename
+import os
+import uuid
 from pymongo import MongoClient
 import json
+from hurry.filesize import size, si
 
 app = Flask(__name__)
+app.config['UPLOAD_FOLDER'] = '../Uploads'
 
 client = MongoClient('localhost', 27017)
```

Comment:
Do not hard code the DB host and port, pass it by a .env file.

Reply:
Sure, I will try that.